STYLEGUIDE_CONVENTION.md

```diff
@@ -0,0 +1,199 @@
+# Code style guide
+
+## Table of Contents
+
+1. [React](#react)
+2. [TypeScript](#typescript)
+3. [CSS](#css)
+
+## React
+
+### JSX
+
+Prefer normal functions (non arrow functions or classes):
+
+```JSX
+// bad
+class Listing extends React.Component {
+  render() {
+    return <div>{this.props.hello}</div>;
+  }
+}
+
+// bad (relying on function name inference is discouraged)
+const Listing = ({ hello }) => (
+  <div>{hello}</div>
+);
+
+// good
+function Listing({ hello }) {
+  return <div>{hello}</div>;
+}
+```
+
+### Extract Login from jsx
```

Comment:
i didn't understand this topic and the title. please rewrite the title and include a brief explanation.